### ISSUE TITLE
Problems after removal of navbar.html

### ISSUE THREAD

**Issue by per-hed** [LotarProject/dokuwiki-template-bootstrap3#128]
For our Dokuwiki-implementation we use the bootstrap3-template with a custom navbar.html as follows:

```php
<?php if (! empty($_SERVER['REMOTE_USER'])): ?>

Anything in the navbar should only be visible when logged in.

  <?php global $INFO; ?>
  <?php if ($INFO['isadmin'] === TRUE || $INFO['ismanager'] === TRUE): ?>

Everything here is only visible to admins + editors (managers).

  <?php endif; ?>
<?php endif; ?>
```
Now, after commit d8091d4735235670cfbc047985fee7a4803771b0, the support for navbar.html is gone. I'm not quite sure how the above PHP-code (or a Dokuwiki-syntax equivalent) may be implemented in the new :navbar scheme?

Is it possible to restore navbar.html support, at least as an option?

Thanks in advance!

**Comment by LotarProject** [comment]
Hi! The solution is to revert the commit and create a new config (eg. ``useLegacyNavbar``) to enable this hook.

At your disposal
Joseph

**Comment by polyzen** [comment]
Shouldn't this be possible with :navbar and ACLs?

**Comment by LotarProject** [comment]
I think with ACL solve the problem, but the above snippet is particular and have two nested conditions.
The new config (``useLegacyNavbar``) is only for the users that using this legacy-navbar with particular code.

**Comment by polyzen** [comment]
Oh, I missed that

**Comment by LotarProject** [comment]
Solved with c7d42cf commit
